mxcube3/routes/qutils.py

```diff
@@ -5,7 +5,7 @@
 import redis
 import itertools
 import xml.etree.ElementTree as et
-
+import logging
```

Comment:
logging is never used in this file

Reply:
it was before I merge with master, removed